gfx/layers/client/TextureClient.cpp

```diff
@@ -1311,6 +1320,9 @@ already_AddRefed<TextureClient> TextureClient::CreateForRawBufferAccess(
     gfx::IntSize aSize, gfx::BackendType aMoz2DBackend,
     LayersBackend aLayersBackend, TextureFlags aTextureFlags,
     TextureAllocationFlags aAllocFlags) {
+  // https://github.com/RecordReplay/backend/issues/5113
+  recordreplay::RecordReplayAssert("TextureClient::CreateForRawBufferAccess Start");
```

Comment:
Yeah, I'm not as concerned about the parameter values here because I don't think they're likely to vary, or to affect the control flow significantly beyond what we can determine using the other asserts.  If we still see mismatches which point to variations in these parameters we can certainly improve the assertions though.